Application/src/Parser.java

```diff
@@ -0,0 +1,135 @@
+import Model.diagramObject;
+import javafx.scene.control.Alert;
+import com.google.gson.Gson;
+
+
+/**
+ * @author Rashad Kamsheh & Isabelle Törnqvist
+ * @since 2017-10-16
+ *
+ * made with usage of Gson
+ * Gson is released under the Apache 2.0 license.
+
+    Copyright 2008 Google Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Does this fit with our license?

Reply:
+1

Reply:
@timjon Yes!
as long as you make:
1- Licence and copyright notice
2- State the changes the you make